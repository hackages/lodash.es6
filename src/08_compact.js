/**
 * Creates an array with all falsey values removed. The values false, null, 0, "", undefined, and NaN are falsey.
 */
export function compact(array) {}
