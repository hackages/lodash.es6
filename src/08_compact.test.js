import {compact} from './08_compact'

describe('compact', () => {
  it('should compact', () => {
    expect(
      compact([
        1,
        2,
        false,
        {a: 32},
        NaN,
        null,
        undefined,
        false,
        null,
        0,
        '',
        undefined,
      ])
    ).toMatchSnapshot()
  })
})
