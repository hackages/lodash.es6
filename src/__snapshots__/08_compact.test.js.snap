// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compact should compact 1`] = `
Array [
  1,
  2,
  Object {
    "a": 32,
  },
]
`;
